scripts/check.sh

```diff
@@ -29,7 +29,8 @@ cargo check --quiet  --all-targets
 cargo check --quiet  --all-targets --all-features
 cargo check --quiet  -p egui_demo_app --lib --target wasm32-unknown-unknown
 cargo check --quiet  -p egui_demo_app --lib --target wasm32-unknown-unknown --all-features
-cargo test  --quiet --all-targets --all-features
+# TODO(lucasmerlin) re-enable --all-features once the tests work with the unity feature
```

Comment:
We should consider doing something like this so we can find the relevant places in the code from the issue number. We do this in Rerun, and it's quite useful. The GitHub extension for vs-code even correctly interprets it as an issue number to the right repository, with a preview on hover (!)

```suggestion
# TODO(#5297) re-enable --all-features once the tests work with the unity feature
```

Reply:
Would be awesome if this could also automatically show up in the relevant issue, like it does when you mention an issue in a comment. But I guess they don't because that could have a lot of false positives?